softee220_week02/api-practice.py

```
@@ -0,0 +1,111 @@
+
+# # 신문기사 데이터 추출 및 CSV 저장
+# 이 노트북에서는 GDELT API를 사용하여 특정 키워드에 맞는 신문기사를 추출하고, Newspaper3k 라이브러리를 사용해 각 기사의 본문을 파싱한 후, 이를 CSV 파일로 저장하는 방법을 설명합니다.
+
+# ### 1. 필요한 라이브러리 설치 및 임포트
+# 
+# 이 단계에서는 필요한 라이브러리들을 설치합니다. gdeltdoc는 GDELT API를 사용하여 기사를 검색할 때 필요하고, newspaper3k는 각 기사에서 본문을 추출할 때 사용됩니다. 또한, pandas 라이브러리를 사용해 데이터를 쉽게 다룰 수 있습니다.
+
+# In[2]:
+
+
+get_ipython().system('pip install gdeltdoc')
```

Ruff (0.13.1)

Undefined name `get_ipython`

(F821)